src/navigation.ts

```diff
@@ -24,6 +24,22 @@ import {
 } from './workspace_utilities';
 import {PassiveFocus} from './passive_focus';
 
+/**
+ * The default coordinate to use when focusing on the workspace and no
+ * blocks are present. In pixel coordinates, but will be converted to
+ * workspace coordinates when used to position the cursor.
+ */
+const DEFAULT_WS_COORDINATE: Blockly.utils.Coordinate =
+  new Blockly.utils.Coordinate(100, 100);
+
+/**
+ * The default coordinate to use when moving the cursor to the workspace
+ * after a block has been deleted. In pixel coordinates, but will be
+ * converted to workspace coordinates when used to position the cursor.
+ */
+const WS_COORDINATE_ON_DELETE: Blockly.utils.Coordinate =
```

Comment:
Moved from the class, but not sure if this was intended to be mutable as a sort of configuration.

Reply:
I suspect this will need to be configurable in the final version, but I think for now this makes sense as a const here.